python/lsst/ts/imsim/imsimCmpt.py

```diff
@@ -651,7 +653,7 @@ def mapOpdDataToListOfWfErr(self, opdZkFileName, sensorIdList, sensorNameList):
             sensorWavefrontData = SensorWavefrontError(numOfZk=self.numOfZk)
             sensorWavefrontData.sensorId = sensorId
             sensorWavefrontData.sensorName = sensorName
-            sensorWavefrontData.annularZernikePoly = opdZk[sensorId]
+            sensorWavefrontData.annularZernikePoly = opdZk[sensorId] / 1e3
```

Comment:
I guess this converts nm to um - why was it not there before? Does the addition of OPD only mode mean that one needs to proceed to um?

Reply:
Yes,  imSim outputs OPD in nanometers so need to change to microns to be consistent with what WEP expects. Before we didn't need to interface OPD results with WEP because we could just do it when making plots but now we need to feed in OPD to WEP so need to do the conversion here in the code. I'll add a comment on this.